packages/source_highlight.rb

```diff
@@ -6,25 +6,25 @@
 class Source_highlight < Autotools
   description 'Convert source code to syntax highlighted document'
   homepage 'https://www.gnu.org/software/src-highlite/'
-  version '3.1.9-e4cf32d-2'
+  version '3.1.9-98bcd3f'
```

Comment:
This is a different git tag than the git_hashtag

Reply:
Also, from the home page, you could also add ctags support.  Since version 2.2, source-highlight can also generate cross references; in order to do this it relies on GNU Ctags, http://ctags.sourceforge.net/.

`depends_on 'ctags'`

Reply:
Rebuilding `ctags`, but this should also automatically be part of the build since it is in build-deps...

Reply:
> Also, from the home page, you could also add ctags support. Since version 2.2, source-highlight can also generate cross references; in order to do this it relies on GNU Ctags, http://ctags.sourceforge.net/.
> 
> `depends_on 'ctags'`

Rebuilt with newer `ctags` on the system.